conanfile.py

```diff
@@ -7,13 +7,15 @@
 class MongoCDriverConan(ConanFile):
     name = "mongo-c-driver"
     version = "1.9.4"
-    url = "http://github.com/bincrafters/conan-mongo-c-driver"
-    description = "A high-performance MongoDB driver for C "
-    license = "https://github.com/mongodb/mongo-c-driver/blob/{0}/COPYING".format(version)
+    url = "https://github.com/mongodb/mongo-c-driver"
+    description = "A high-performance MongoDB driver for C"
+    topics = ["conan", "libmongoc", "mongodb"]
```

Comment:
Really sorry for this confusion, but the topics need another update.

Turned out the documentation wasn't correct about this attribute.

https://github.com/conan-io/docs/pull/935

https://github.com/bincrafters/conan-templates/commit/e0eca7e796ff3ae8ac7a82abb63463b801efdff9#diff-2b1d42f71f22b7ea0412d7602dec166fR13

Reply:
```suggestion
    topics = ("conan", "libmongoc", "mongodb")
```